### PR TITLE
python eclasses: remove old EAPI support

### PR DESCRIPTION
**Pull request by mgorny**
Should require only cleaning up old Portage.

### CODE REVIEW

File: eclass/python-utils-r1.eclass [hunk 1]
```diff
@@ -1169,17 +1160,14 @@ python_fix_shebang() {
 		done < <(find -H "${path}" -type f -print0 || die)
 
 		if [[ ! ${any_fixed} ]]; then
-			local cmd=eerror
-			[[ ${EAPI} == 5 ]] && cmd=eqawarn
-
-			"${cmd}" "QA warning: ${FUNCNAME}, ${path#${D%/}} did not match any fixable files."
+			eerror "QA warning: ${FUNCNAME}, ${path#${D%/}} did not match any fixable files."
```
Comment (thesamesam) [inline eclass/python-utils-r1.eclass:1163]: Not eqawarn?

**Review by thesamesam — APPROVED** [review]
On mobile so I wasn’t able to super easily check for other possible cleanups but changes look good.

File: eclass/python-utils-r1.eclass (hunk 1 above)
Comment (mgorny) [reply eclass/python-utils-r1.eclass:1163]: Nope, it's a fatal warning now ;-).

Comment (mgorny) [reply eclass/python-utils-r1.eclass:1163]: I've changed it to 'QA error'.

**Pull request closed without merging**